pkg/policy/api/entity.go

```diff
@@ -141,11 +141,4 @@ func InitEntities(clusterName string, treatRemoteNodeAsHost bool) {
 		endpointSelectorKubeAPIServer,
 		NewESFromLabels(labels.NewLabel(k8sapi.PolicyLabelCluster, clusterName, labels.LabelSourceK8s)),
 	}
-
-	hostSelectors := make(EndpointSelectorSlice, 0, 2)
-	hostSelectors = append(hostSelectors, endpointSelectorHost)
-	if treatRemoteNodeAsHost {
-		hostSelectors = append(hostSelectors, endpointSelectorRemoteNode)
-	}
-	EntitySelectorMapping[EntityHost] = hostSelectors
```

Comment:
If I'm not mistaken, we should still keep `endpointSelectorHost` here, i.e.

```go
EntitySelectorMapping[EntityHost] = EndpointSelectorSlice{endpointSelectorHost}
```

Reply:
`EntitySelectorMapping[EntityHost]` is already filled in the global variable here https://github.com/cilium/cilium/blob/main/pkg/policy/api/entity.go#L95. 
This part of the code was only present to handle the remote node as a host usecase, no need to keep it.